exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hour: 0, minute: 0
+
+  @doc """
+  Returns a string representation of a clock:
+
+      iex> Clock.new(8, 9) |> to_string
+      "08:09"
+  """
+  @spec new(integer, integer) :: Clock
```

Comment:
I'm not 100% sure this typespec is valid without defining a type for this module - is it? I'd be happy to see `Clock.t`, but I could also just be wrong on this. I just haven't seen this type of syntax in the standard library, so it struck me as something we might want to look at further.